sydent/http/servlets/infoservlet.py

```diff
@@ -86,20 +89,32 @@ def render_GET(self, request):
 
         result = copy.deepcopy(result)
 
-        # Remove 'requires_invite' if found
-        if 'requires_invite' in result:
-            result.pop('requires_invite')
+        if not self.internal:
```

Comment:
so I'm afraid I was careful *not* to suggest merging the two into a single servlet for a reason. Half of the servlet is now conditional on `if not self.internal`, which is confusing and redundant.

Please keep them separate, but factor the common code out to a common function.

Reply:
Erk, ok, confused handler and servlet. Not sure where to put this combined function though. In one of the servlets which the other servlet imports?

Reply:
Going with the above.